src/commands/notify.yml

```diff
@@ -8,6 +8,10 @@ parameters:
     description: "IRC server port. Defaults to 6667."
     type: integer
     default: 6667
+  pass:
+    description: "The IRC server password. Defaults to the environment variable 'IRC_SERVER_PASSWORD'."
+    type: string
+    default: ${IRC_SERVER_PASSWORD}
```

Comment:
Almost :) We do have an actual `env_var` type, that removes the need for the `${}` bit. https://circleci.com/docs/2.0/reusing-config/#environment-variable-name

Reply:
Well it removes the need for the `${}` part there, but requires it on actual use.
Updated.

I did not change the definiiton of `IRC_SERVER` though, as this is essentially unrelated to this PR